go/registry/api/api.go

```diff
@@ -1403,6 +1403,9 @@ type ConsensusParameters struct {
 	// disabled outside of the genesis block.
 	DisableKeyManagerRuntimeRegistration bool `json:"disable_km_runtime_registration,omitempty"`
 
+	// EnableKeyManagerCHURP is true iff the CHURP extension for the key manager is enabled.
+	EnableKeyManagerCHURP bool `json:"enable_km_churp,omitempty"`
```

Comment:
Should this rather be something like a list of enabled KM extension identifiers (`EnableKeyManagerExtensions`)? To not introduce one flag per extension.

Reply:
Could do that. But my plan was to remove this after the upgrade.